### PR TITLE
agent: Add missing nsenter import

### PR DESCRIPTION
**Pull request by vbmade2000**
We need to import `nsenter` package to make `init` function to be able to spawn a
new container.

Fixes: #95 

Signed-off-by: Malhar Vora <mlvora.2010@gmail.com>

### CODE REVIEW

**Comment by sameo** [comment]
LGTM

File: agent.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ import (
 	"syscall"
 	"time"
 
+    _ "github.com/opencontainers/runc/libcontainer/nsenter"
```
Comment (sboeuf) [inline agent.go]: This is not indented the right way, and it's not in the right place in the code I think.
Could run some linters to fix that please:
```bash
go_packages=$(go list ./... | grep -v protocols | grep -v vendor)
go_files=`go list -f  '{{.Dir}}/*.go' $go_packages`
go fmt $go_packages
golint $go_packages
go vet $go_packages
gofmt -s -w $go_files
gocyclo -over 15 $go_files
ineffassign .
```

Comment (vbmade2000) [reply agent.go]: Sure. Quickly.

**Comment by vbmade2000** [comment]
@sboeuf If you still see some issue and in rush then I can send you patch.

**Comment by sboeuf** [comment]
@vbmade2000 thx for the quick fix !

**Comment by sameo** [comment]
@vbmade2000 Thanks for your contribution, merging now.

**Pull request merged**